src/app/components_ngrx/work-item-cell/work-item-cell.component.less

```diff
@@ -23,3 +23,10 @@
     margin: 0 !important;
   }
 }
+.f8-planner-cell__state {
+  display: flex;
+  &-text {
+    line-height: normal;
+    white-space: normal;
+  }
+}
```

Comment:
After applying these properties all the other align vertically top. Because text is vertically aligned middle by line height. So, this should be 
```
.f8-planner-cell__state {
  display: table;
  &-text {
    line-height: normal;
    white-space: normal;
    display: table-cell;
    vertical-align: middle;
  }
```